pdp/contract/PDPVerifier.go

```diff
@@ -197,6 +197,37 @@ func (_PDPVerifier *PDPVerifierTransactorRaw) Transact(opts *bind.TransactOpts,
 	return _PDPVerifier.Contract.contract.Transact(opts, method, params...)
 }
 
+// EXTRADATAMAXSIZE is a free data retrieval call binding the contract method 0x029b4646.
```

Comment:
this is a different ABI to the 3.0.0 that got tagged where these were removed: https://github.com/FilOzone/pdp/commits/v3.0.0/src/PDPVerifier.sol

I guess you need to update/sync something?